### PR TITLE
/teams improvements

### LINKED ISSUE #6826: [CAL-975] /teams: recommended next steps
CleanShot 2023-02-01 at 10 09 37@2x

figma: https://www.figma.com/file/ZJNAFHCAq6rnuGRgUFxjzQ/Cal.com-Teams?node-id=709%3A114599&t=5j1V4RlrCIrWXZpK-4

<sub>[CAL-975](https://linear.app/calcom/issue/CAL-975/teams-recommended-next-steps)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
- [x] fixed empty screen text for teams
- [x] new "recommended steps" for teams
- [x] ~~wait for managed events to be finished (requires managed=true in param)~~
- [x] show "appearance" instead of managed events for now 
- [ ] figure out why the empty screen doesnt show when you delete the last team (requires a hard reload)

CleanShot 2023-02-01 at 21 15 20@2x

https://user-images.githubusercontent.com/8019099/216069269-16a36f1a-0dca-4ce4-abcb-d92be258e451.mp4

### CODE REVIEW

**Comment by Jaibles** [comment]
>  wait for managed events to be finished (requires managed=true in param)

Could we have the button set to disabled with the label “coming soon” instead of waiting?

**Comment by PeerRich** [comment]
> > wait for managed events to be finished (requires managed=true in param)
> 
> Could we have the button set to disabled with the label “coming soon” instead of waiting?

showing appearance now

File: packages/ui/components/card/Card.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
+// @TODO: turn this into a more generic component that has the same Props API as MUI https://mui.com/material-ui/react-card/
 import Link from "next/link";
 import { ReactNode } from "react";
 import React from "react";
 
 import classNames from "@calcom/lib/classNames";
+import { FiArrowRight, FiCalendar, FiPlus } from "@calcom/ui/components/icon";
```
Comment (emrysal) [inline packages/ui/components/card/Card.tsx:7]:
```suggestion
import { FiArrowRight } from "@calcom/ui/components/icon";
```

**Review by emrysal — APPROVED** [review]
Looks good, lgtm

**Comment by emrysal** [comment]
Do we want to make the "Recommended next steps" dismissible? (Just checked the designs and it has an X)

**Comment by PeerRich** [comment]
@emrysal question is, why does the emptyscreen not come when I delete the last team. it currently requires a reload

**Pull request merged**